Artifacts/windows-deploymentgroup/InstallAndConfigureDeploymentGroupAgent.ps1

```diff
@@ -144,6 +144,76 @@ function Configure-DeploymentGroupAgent
 
     if ($runAsAutoLogon)
     {
+        # Create a PS session for the user to trigger the creation of the registry entries required for autologon
+        if ($windowsLogonAccount.Split("\").Count -eq 2)
```

Comment:
This change means that we can no longer use an email address to authenticate. Am I to assume that's expected?

Reply:
You either provide a local account or a domain joined account that exists on the vm. There is an else block that handles non domain accounts. The behavior has not changed and both cases are handled. When you say email address you mean a domain joined account right?